website/pages/en/firehose/[[...slug]].mdx

```diff
@@ -38,45 +33,27 @@ export async function getStaticProps({ params: { slug = ['README'] } }) {
     // remove gitbook {% ... %} elements
     .replaceAll(/{%.*?%}/g, '')
     // Replaces all the relative paths of images to absolute paths to the repo
-    .replaceAll('../assets', 'https://raw.githubusercontent.com/streamingfast/firehose-docs/master/assets/')
+    .replaceAll('../assets', `https://raw.githubusercontent.com/${user}/${repo}/${branch}/assets/`)
     // close img tags only if he doesn't point to .gitbook
     .replaceAll(/<img(.*?)>/g, (...m) => (m[1].includes('src=".gitbook') ? '' : `<img${m[1]}/>`))
-  let mdx = await buildDynamicMDX(data, {
+  const mdx = await buildDynamicMDX(data, {
     mdxOptions: {
-      format: 'mdx',
-      remarkPlugins: [
-        () => (tree, _file, done) => {
-          // enhance links
-          visit(tree, 'link', (node) => {
-            if (node.url.startsWith('./')) {
-              node.url = node.url.slice(2)
-            }
-            if (node.url.startsWith('/')) {
-              // (foo)[/foo/bar]
-              node.url = node.url.replace('/', '/firehose/')
-            } else if (!node.url.includes('/') && node.url.endsWith('.md')) {
-              // (foo)[foo.md]
-              node.url = [...slug.slice(0, -1), node.url].join('/')
-            }
-          })
-          done()
-        },
```

Comment:
@saihaj This seems an old solution that was refactored by @benface

I extracted function from substreams docs to reuse here, but some links are broken, could you take a look why? maybe due to excluding files that you made?

Reply:
https://github.com/graphprotocol/docs/pull/399/commits/cf5b96f4ca282be2fbaaccca57b47599c5c003cc